index.html

```diff
@@ -29,22 +30,205 @@ <h1>
         don’t hesitate to contact me.
       </p>
     </div>
-  </section>
+  </div>
 
-  <section id="footer">
     <div class="social-links">
       <ul>
-        <li><a href="#"><img src="twitter (1).svg"></a></li>
-        <li><a href="#"><img src="Linkedin icon.svg"></a></li>
-        <li><a href="#"><img src="medium.svg"></a></li>
-        <li><a href="#"><img src="github.svg"></a></li>
-        <li><a href="#"><img src="angellist.svg"></a></li>
+        <li><a href="#"><img src="images/twitter (1).svg"></a></li>
+        <li><a href="#"><img src="images/Linkedin icon.svg"></a></li>
+        <li><a href="#"><img src="images/medium.svg"></a></li>
+        <li><a href="#"><img src="images/github.svg"></a></li>
+        <li><a href="#"><img src="images/angellist.svg"></a></li>
```

Comment:
- [OPTIONAL] I noticed that the first icon in your social link is not displaying in the browsers, please look into this. 👍 
- Screenshot: 
![image](https://user-images.githubusercontent.com/107632780/229313892-a550b949-d98d-42b9-85f2-3213e57004ce.png)
- - Also I noticed that your image tags do not have the `alt` attribute please consider adding them because when an image cannot be displayed. The value of the attribute can be read by screen readers. 😄